psalm-baseline.xml

```diff
@@ -117,18 +117,43 @@
       <code><![CDATA[$key]]></code>
     </PossiblyNullArgument>
   </file>
+  <file src="test/AbstractFactory/ReflectionBasedAbstractFactoryTest.php">
+    <PossiblyUnusedMethod>
+      <code><![CDATA[classNamesWithoutConstructorArguments]]></code>
+      <code><![CDATA[invalidRequestNames]]></code>
+    </PossiblyUnusedMethod>
+  </file>
   <file src="test/AbstractPluginManagerTest.php">
-    <MixedInferredReturnType>
-      <code><![CDATA[array]]></code>
-    </MixedInferredReturnType>
     <MixedPropertyAssignment>
       <code><![CDATA[$instance]]></code>
     </MixedPropertyAssignment>
+    <PossiblyUnusedMethod>
+      <code><![CDATA[shareByDefaultSettings]]></code>
+    </PossiblyUnusedMethod>
+  </file>
+  <file src="test/Command/AheadOfTimeFactoryCreatorCommandTest.php">
+    <PossiblyUnusedMethod>
+      <code><![CDATA[invalidFactoryTargetPaths]]></code>
+    </PossiblyUnusedMethod>
+    <UndefinedAttributeClass>
+      <code><![CDATA[CoversClass]]></code>
+    </UndefinedAttributeClass>
   </file>
   <file src="test/Command/ConfigDumperCommandTest.php">
     <PossiblyUnusedMethod>
       <code><![CDATA[ignoreUnresolvedArguments]]></code>
     </PossiblyUnusedMethod>
+    <UndefinedAttributeClass>
```

Comment:
Why the `UndefinedAttribute…` for `CoversClass` 🤔

Reply:
I've added `PHPUnit\Framework\Attributes\CoversClass` to use statement https://github.com/laminas/laminas-servicemanager/pull/248/commits/7a4e3643d77ecceedde3191d7e0b1aaba74d0db9 👍